### ISSUE TITLE
make new release for magma v2.0 (comparison)

### ISSUE THREAD

**Issue by oconnellm** [NREL/MAGMA#87]

**Comment by claytonpbarrows** [comment]
Given the versioning issues that we have been having, we may want to move towards releasing this as an r package. ... just a suggestion

**Comment by bradystoll** [comment]
huh, that's an interesting thought. In general, I think we should wait a sec before releasing an official version since we are still finding small bugs here and there. Since there are a lot of people starting to use it now, maybe give them a couple weeks to let us know of any issues they find before doing a release

**Comment by claytonpbarrows** [comment]
Agreed, and no rush on the Package idea, but I think it might be a nice way for people to stick with with the version that works with their environement/workflow until they are ready to update to the latest versions of all the packages.
